addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComRfyMessage.java

```diff
@@ -185,11 +180,11 @@ public State convertToState(RFXComValueSelector valueSelector) throws RFXComExce
             if (valueSelector == RFXComValueSelector.COMMAND) {
 
                 switch (command) {
-                    case CLOSE:
```

Comment:
That's a message-internal Command enum, not an OH-global command :-). They're commands from the rfxcom messages. I just changed OPEN/CLOSE to UP/DOWN for consistency with the SDK (but inconsistent with vertical blinds, I guess :-( ).

I'd give it a days or two since there was a recent ping. OTOH it's been a year with no other interest logged so...